TestClasses/TestJson.cls

```diff
@@ -0,0 +1,13 @@
+@isTest
+public class TestJson 
+{
+    @isTest
+    public static void testJson()
+    {
+        String soql = 'SELECT Name FROM Account LIMIT 10';
```

Comment:
If account is not create then result is always zero. 
Never use org data in unit test method.
assert the size of list of accounts that retrieve through SOQL.